### PR TITLE
[Blog]Example of binlog load usage

### PR DESCRIPTION
**Pull request by hf200012**
Example of binlog load usage

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by xy720 — COMMENTED** [review]
Thank you for providing examples

File: docs/zh-CN/article/articles/doris-binlog-load.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Apache Doris Binlog Load使用方法及示例
+
+Binlog Load提供了一种使Doris增量同步用户在Mysql数据库的对数据更新操作的CDC(Change Data Capture)功能，使用户更方面的完成Mysql数据的导入
```
[the remark this answers is not in the document]
Comment (hf200012) [reply docs/zh-CN/article/articles/doris-binlog-load.md:34]: I didn't see the difference between these two sentences

File: docs/zh-CN/article/articles/doris-binlog-load.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+![image-20211110160710709](/images/binlog/image-20211110160710709.png)
+
+#### 3.1.6 多表同步
+
+多表同步只需要像下面这样写法就可以了
+
+```sql
+CREATE SYNC test_2.doris_mysql_binlog_demo_job 
+(
+	FROM demo.test_cdc INTO doris_mysql_binlog_demo,
```
[the remark this answers is not in the document]
Comment (hf200012) [reply docs/zh-CN/article/articles/doris-binlog-load.md]: OK

File: docs/zh-CN/article/articles/doris-binlog-load.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (morningman) [reply docs/zh-CN/article/articles/doris-binlog-load.md:34]: 更方便，不是“更方面”

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**